### DISCUSSION TITLE
Missing metrics failures&successes in statsd

### DISCUSSION THREAD

**Discussion by boryakbrk** [apache/airflow#29415] [Q&A]
Hi!
I'm using airflow 2.5 deployed on kubernetes. Having opened the exporter in statsd, I cannot find the metrics of a successful DAG execution or an error that interest me.
`airflow_dagrun_duration_count`
`airflow_dagrun_failed_count`
All other metrics are present in statsd. Help me figure out why this is happening.

**Comment by hussein-awala** [comment]
Airflow doesn't emit the count of failed and succeed dag runs, [here](https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/logging-monitoring/metrics.html) is the list of available metrics, it emits `dagrun.duration.success.<dag_id>` and `dagrun.duration.failed.<dag_id>` which you can use to get the count (`count(metric)`)
